app/views/layouts/design_system.html.erb

```diff
@@ -1,5 +1,11 @@
 <% environment = GovukPublishingComponents::AppHelpers::Environment.current_acceptance_environment %>
 
+<% if user_signed_in? && current_user.has_email? %>
+  <% content_for :head do %>
+    <meta name="custom-dimension:8" content="<%= current_user.organisation_slug.present? ? current_user.organisation_slug : '(not set)' %>">
```

Comment:
I think it should be possible to shorten this by using [Object#presence](https://apidock.com/rails/Object/presence):

```suggestion
    <meta name="custom-dimension:8" content="<%= current_user.organisation_slug.presence || '(not set)' %>">
```